packages/styleguide/src/templates/shared/rules/embedDatawrapperRule.js

```diff
@@ -0,0 +1,21 @@
+import { matchZone } from '@republik/mdast-react-render'
+import Datawrapper from '../../../components/Datawrapper'
+
+export const embedDataWrapperRule = (resizable = true) => ({
```

Comment:
Can we make that an object, so it becomes `embedDataWrapperRule({ resizable: false })` to make it more obvious what this argument does?

Reply:
we can. regarding your question with respect to size/resizing. it is allowed for figures in the article-as-email template, which is why i decided to keep it consistent and allow it there. Editorial newsletter don't allow resizable charts.

Reply:
Yes, when the charts are rendered in the browser, DW takes care of including the alt text. That's why I removed it from the modal. To use the same text on the images would require us to load the embed.json of the chart and set it on the image … not sure where to do that in this module maze.

Reply:
OK, we can also keep it. And in the article emails the size will be ignored?

Reply:
editorial newsletter, no resize:
- single sized chart (450 or 600px, depending what we decide)

article emails, two sizes for charts:
- regular chart (450/600px wide)
- edge-to-edge (like article-email figure, 1280px wide)